content/en/getting_started/synthetics/browser_test.md

```diff
@@ -0,0 +1,76 @@
+
+## Configure your test
+
+1. In the Datadog application, hover over **[UX Monitoring][2]** in the left hand menu and select **Synthetics Test**. 
+2. In the top right corner, click the **New Test** button. 
+3. Select **Browser Test**.
+4. Define the configuration of your browser test:
+    - Add the URL of the website you’re interested in monitoring. If you don’t know what to start with, you can use https://www.shopist.io/ as a test web application.”
+    - Name the test.
+    - You can set tags such as `prod` and `shopist` to your test. It will help to keep things organized and allow you to quickly find the tests you're interested in when on the homepage.
+    - Choose devices and locations for testing. In this example, the test is only run on **Large Laptop** & on **English speaking countries**.
+    - Specify a test frequency.
+    - Set alert conditions to determine the circumstances under which you want a test to send a notification alert. To avoid being alerted on network blips that might happen on specific locations, this test is configured as:
+        `An alert is triggered if your test fails for 0 minutes from any 3 of 13 locations`
+    - Set alert conditions to determine the circumstances under which you want a test to send a notification alert. 
+      - If you want to avoid being alerted on network blips that might happen on isolated locations, you can for instance set:
+        `An alert is triggered if your test fails for 0 minutes from any 3 of 13 locations`
+      - If you want to be sure that a test execution is only considered a failure after 2 failed test runs, specify how many times you want your test to be re-run before being actually considered a failure:
+        `Retry 1 time before location is marked as failed`
+    - Write an alert message and specify which email addresses should be notified when the alert is triggered. No additional set up is required to start receiving alert emails from Datadog. You can also use [integrations][3], such as Slack, Pagerduty, webhooks, etc., to receive alert notifications.
+    - Click **Save & Edit Recording**.
+
+{{< img src="getting_started/synthetics/configured-browser-test.gif" alt="Configured browser test" responsive="true" style="width:90%;">}}
+
+## Record your test steps
+
+Once the test configuration is saved, Datadog will prompt you to download the [Datadog test recorder][4] extension. Browser tests can be only recorded on **[Google Chrome][5]**. Download and install the extension.
+
+Once this extension is installed, begin recording your test steps by clicking the **Start Recording** button. Navigate your page in the iframe to the right of the recording options. When you select a div, image, or any area of your page, the actions are recorded and used to create steps within the browser test. You can record the uploading of files as an action, as outlined in the [browser test actions doc][6].
```

Comment:
Let's completely remove the reference to the ability to upload files, and just mention the other contains all the available steps.